### PR TITLE
The Return of the Classic Technosuit

### PR DESCRIPTION
**Pull request by Honestly101**
## About The Pull Request

Brings back the classic technomancer voidsuit and adds the femboy suit to the maint loot table.

## Why It's Good For The Game

People have been saying that the new technosuit is ugly and makes you look like a femboy with a skirt and huge honkers and that the old void suit in general looks so much better.

## Changelog
:cl:
tweak: Reverted the icon state of the technomancer voidsuit.
add: Added the new technomancer voidsuit to the maint loot table.
/:cl:

### CODE REVIEW

**Comment by Medcar** [comment]
the return of the king

**Comment by mattroks101** [comment]
"code: changed some code"
Real changelog please.

**Comment by MadmanMartian** [comment]
I'd like to use the techno-classic suit for something else, if possible.

**Comment by Wouju** [comment]
> I'd like to use the techno-classic suit for something else, if possible.

maybe we could make a new sprite that's just the classic one edited, similary how the TE hardsuit (more specifically the helmet) looks similar to the voidsuit

A lot of people complained about the techno voidsuit and want the old/classic one to come back

**Review by genessee596 — CHANGES_REQUESTED** [review]
make new techno voidsuit spawn in maintenance, old one can come back as main techno one

**Comment by Honestly101** [comment]
yes i agree with that idea, but i dont know how maint/trash spawning works. i only do edits of what already exists, i ain't even a real coder, so if anyone can spare some time to do it, that would be nice.

**Comment by Evankhell561** [comment]
> yes i agree with that idea, but i dont know how maint/trash spawning works. i only do edits of what already exists, i ain't even a real coder, so if anyone can spare some time to do it, that would be nice.

`/obj/item/clothing/head/space/void/engineering/alt
	spawn_blacklisted = FALSE`

**Comment by Honestly101** [comment]
I thought so, but I assume I'll have to make a second thing for the fem it suit so they will be separate objects.

**Comment by Honestly101** [comment]
Ill get to doing all of it when I get back to my desktop, I'm currently out of town for a few days.

**Comment by Honestly101** [comment]
The entire southwest region of Australia is currently in a lockdown so I won’t be able to work on it for a week

**Comment by Honestly101** [comment]
this is the second void suit is it all correct?

**Comment by Honestly101** [comment]
ok the femboy suit now spawns in maints

**Comment by Honestly101** [comment]
I compiled it and play tested to make sure it all works

**Comment by Honestly101** [comment]
made small changes to the spawn limit and rate of the femboy suit to be more rare

**Comment by Honestly101** [comment]
damn its conflicting

**Comment by Wouju** [comment]
Wait, is it intended for the cursed voidsuit to be a child of the basic void suit (as seen here: https://github.com/discordia-space/CEV-Eris/blob/3dc31de05e3ce61a634379a2189f18a6ab7c7a97/code/modules/clothing/spacesuits/void/void.dm) ?

File: code/modules/clothing/spacesuits/void/station.dm [hunk 1]
```diff
@@ -44,9 +95,9 @@
 		usr.update_action_buttons()
 		return 1
 
-/obj/item/clothing/suit/space/void/engineering
-	name = "Technomancer voidsuit"
-	desc = "A special suit that protects against hazardous, low pressure environments. Has radiation shielding and extra plating."
+/obj/item/clothing/suit/space/void/engineeringcursed
+	name = "Cursed technomancer voitsuit"
```
Comment (Firefox13) [inline code/modules/clothing/spacesuits/void/station.dm:99]:
```suggestion
	name = "outdated Technomancer voidsuit"
```

File: code/modules/clothing/spacesuits/void/station.dm [hunk 2]
```diff
@@ -44,9 +95,9 @@
...
+	desc = "Same purpose as the standard technomancer voidsuit but was rejected by the empress for being unsightly and making the wearer look like a femboy."
```
Comment (Firefox13) [inline code/modules/clothing/spacesuits/void/station.dm:100]:
```suggestion
	desc = "An outdated Technomancer voidsuit that is nearly identical in all properties to its newer version. Nevethless this design was rejected in favour of more streamlined shapes. Rumors claim there was a different reason to it, but we all stick to this one."
```

File: code/modules/clothing/spacesuits/void/station.dm [hunk 3]
```diff
@@ -44,9 +95,9 @@
 		usr.update_action_buttons()
 		return 1
 
-/obj/item/clothing/suit/space/void/engineering
-	name = "Technomancer voidsuit"
-	desc = "A special suit that protects against hazardous, low pressure environments. Has radiation shielding and extra plating."
+/obj/item/clothing/suit/space/void/engineeringcursed
```
Comment (Firefox13) [inline code/modules/clothing/spacesuits/void/station.dm:98]:
```suggestion
/obj/item/clothing/suit/space/void/engineeringold
```

File: code/modules/clothing/spacesuits/void/station.dm [hunk 4]
```diff
@@ -64,10 +115,10 @@
 		/obj/item/device/t_scanner,
 		/obj/item/weapon/rcd
 	)
-	helmet = /obj/item/clothing/head/space/void/engineering
-	spawn_blacklisted = TRUE
+	helmet = /obj/item/clothing/head/space/void/engineeringcursed
+	spawn_blacklisted = FALSE
 
-/obj/item/clothing/suit/space/void/engineering/equipped
+/obj/item/clothing/suit/space/void/engineeringcursed/equipped
```
Comment (Firefox13) [inline code/modules/clothing/spacesuits/void/station.dm:121]:
```suggestion
/obj/item/clothing/suit/space/void/engineeringold/equipped
```

File: code/modules/clothing/spacesuits/void/station.dm [hunk 5]
```diff
@@ -64,10 +115,10 @@
 		/obj/item/device/t_scanner,
 		/obj/item/weapon/rcd
 	)
-	helmet = /obj/item/clothing/head/space/void/engineering
-	spawn_blacklisted = TRUE
+	helmet = /obj/item/clothing/head/space/void/engineeringcursed
```
Comment (Firefox13) [inline code/modules/clothing/spacesuits/void/station.dm:118]:
```suggestion
	helmet = /obj/item/clothing/head/space/void/engineeringold
```

File: code/modules/clothing/spacesuits/void/station.dm [hunk 6]
```diff
@@ -1,8 +1,59 @@
...
+	spawn_blacklisted = TRUE
+
+/obj/item/clothing/suit/space/void/engineering/equipped
+	boots = /obj/item/clothing/shoes/magboots
+	tank = /obj/item/weapon/tank/jetpack/oxygen
+	accompanying_object = null
+	spawn_blacklisted = TRUE
+
+//Femboy engineering rig
+/obj/item/clothing/head/space/void/engineeringcursed
```
Comment (Firefox13) [inline code/modules/clothing/spacesuits/void/station.dm:54]:
```suggestion
/obj/item/clothing/head/space/void/engineeringold
```

File: code/modules/clothing/spacesuits/void/station.dm [hunk 7]
```diff
@@ -1,8 +1,59 @@
...
+	name = "Cursed Technomancer voidsuit helmet"
```
Comment (Firefox13) [inline code/modules/clothing/spacesuits/void/station.dm:55]:
```suggestion
	name = "outdated Technomancer voidsuit helmet"
```

File: code/modules/clothing/spacesuits/void/station.dm [hunk 8]
```diff
@@ -1,8 +1,59 @@
...
+	desc = "This visor has a few more options in it's shape than its classic counter part."
```
Comment (Firefox13) [inline code/modules/clothing/spacesuits/void/station.dm:56]:
```suggestion
	desc = "This visor has a few more options in its shape than its more newer version."
```

**Review by Firefox13 — CHANGES_REQUESTED** [review]
We all understand how this suit looks, but please have some common sense

**Comment by Honestly101** [comment]
ok so i went back into the spawn paths, changed the names to those that are in Firefox's changes, and returned the value back to normal and it seems to be still conflicting, can i get some help with the issue?

**Comment by Honestly101** [comment]
ok so i compiled it again in dreammaker and run a private server, no errors occured during the compile and all works as intended.

**Comment by hyperioo** [comment]
Did you try the procedure to properly solve the conflicts?

```
Step 1: From your project repository, check out a new branch and test the changes.

git checkout -b Honestly101-master master
git pull https://github.com/Honestly101/CEV-Eris.git master

Step 2: Merge the changes and update on GitHub.

git checkout master
git merge --no-ff Honestly101-master
git push origin master
```

**Comment by Honestly101** [comment]
So make a new branch then remerge it, is that correct?

**Comment by Honestly101** [comment]
sorry i just have a ape brain.

**Comment by hyperioo** [comment]
Yeah that's what those commands do. That way you will be able to solve the conflicts in your editor because apparently GitHub cannot do it online "These conflicts are too complex to resolve in the web editor"/

**Comment by Honestly101** [comment]
ok i i resolved the conflict on the app i do believe and it appears it may have works as the warning is no longer there.

**Comment by hyperioo** [comment]
Oh damn your PR now contains all recents commits (126 commits and 344 files changes) 😨

**Comment by Honestly101** [comment]
oh shit

**Comment by Honestly101** [comment]
holy fuck what have i done to this PR its eldrich fucking horror now

**Comment by hyperioo** [comment]
I'm gonna try something. We should have rebased on master before doing the procedure to avoid including all recent commits.

**Comment by Honestly101** [comment]
should i kill this PR and my fork?

**Comment by Honestly101** [comment]
ok sure

**Comment by Honestly101** [comment]
uhhh i think it will just be better to kill both this PR and my fork, and start from the beginning, i already copied all i need into a note pad, should i do it?

**Comment by Honestly101** [comment]
Just because this shit has death spiralled real fast.

**Comment by Honestly101** [comment]
i have layed this eldrich horror i created to a rest

**Pull request closed without merging**